packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -682,17 +689,23 @@ export default class TradeStore extends BaseStore {
         if (!proposal_info) {
             return;
         }
-        const { contract_type, barrier, high_barrier, low_barrier } = proposal_info;
+        const { barrier, contract_type, high_barrier, low_barrier } = proposal_info;
 
         if (isBarrierSupported(contract_type)) {
             const color = this.root_store.ui.is_dark_mode_on ? BARRIER_COLORS.DARK_GRAY : BARRIER_COLORS.GRAY;
-            const turbos_color = getTurbosColor(contract_type);
+            const turbos_hovered_color = contract_type === 'TURBOSSHORT' ? BARRIER_COLORS.RED : BARRIER_COLORS.GREEN;
```

Comment:
@kate-deriv let's not hardcode TURBOSSHORT here but instead use a helper function which would accept contract_type, incapsulate logics, and return a color based on the contract_type passed into it. 🙂

Reply:
@maryia-deriv I'll write smth like this:
```
const getHoveredColor = type => {
                switch (type) {
                    case 'TURBOSSHORT':
                        return BARRIER_COLORS.RED;
                    case 'TURBOSLONG':
                        return BARRIER_COLORS.GREEN;
                    default:
                        return BARRIER_COLORS.GRAY;
                }
};
```
But I suggest create this function in helpers folder and name it not a `turbos-utils.js`, but a `barrier-utils.js` in order to reuse this logics later, not only for Turbos